core/src/main/java/org/geysermc/geyser/level/block/BlockStateValues.java

```diff
@@ -84,6 +87,55 @@ public final class BlockStateValues {
      * @param blockData      JsonNode of info about the block from blocks.json
      */
     public static void storeBlockStateValues(String javaId, int javaBlockState, JsonNode blockData) {
+        String cleanJavaId = BlockUtils.getCleanIdentifier(javaId);
+        switch (cleanJavaId) {
+            case "minecraft:anvil":
```

Comment:
Could we instead make this a mappings generator thing? I like the set idea still instead of adding it to the BlockMapping class, but this feels like something that's easy to slip up on new versions.

Reply:
It might be possible if we can instantiate a `Level` to call the `use` method on blocks.
However, we may also get false positives for blocks that have special logic in this `use` method.